assets/js/base/components/label/index.js

```diff
@@ -11,10 +11,10 @@ import classNames from 'classnames';
  * specified via props.
  *
  * @param {Object} props Incoming props for the component.
- * @param {string} props.label Label content.
- * @param {string} props.screenReaderLabel Content for screen readers.
- * @param {string} props.wrapperElement What element is used to wrap the label.
- * @param {Object} props.wrapperProps Props passed into wrapper element.
+ * @param {string} [props.label] Label content.
+ * @param {string} [props.screenReaderLabel] Content for screen readers.
+ * @param {string} [props.wrapperElement] What element is used to wrap the label.
+ * @param {Object} [props.wrapperProps] Props passed into wrapper element.
```

Comment:
We had default props for `wrapperProps`:

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/fix/3234-express-tabs/assets/js/base/components/label/index.js#L67-L69

But if I remember correctly, we agreed to use default parameters for functional components instead of default props, so I changed that in c944d9365ad939fab5899f2ad2530f0e8c1eb602.

Then, there is no need to update the JSDoc since it's not a required prop, right?